app/services/create_github_repo_service/entity.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+class CreateGitHubRepoService
+  class Entity
+    def self.build(assignment, collaborator)
+      if collaborator.is_a?(User)
+        Individual.new(assignment, collaborator)
+      else
+        Team.new(assignment, collaborator)
+      end
+    end
+
+    attr_reader :assignment, :collaborator, :organization, :invite_status
+    delegate :status, to: :invite_status
+
+    def initialize(assignment, collaborator)
+      @assignment = assignment
+      @collaborator = collaborator
+      @organization = assignment.organization
+      @invite_status = assignment.invitation.status(collaborator)
+    end
+
+    def repo_name
+      @repo_name ||= generate_repo_name
+    end
+
+    def default_repo_name
+      "#{assignment.slug}-#{slug}"
+    end
+
+    def owner
```

Comment:
This is an organization login, could you make the method name more descriptive?

Reply:
renaming it to `organization_login`